tests/platforms/test_host_collector.py

```diff
@@ -184,6 +194,54 @@ def test_prepare_payload_with_snapshot_disabled_python_packages(self, mock_shoul
         self.assertIn('snapshot', payload['metrics']['plugins'][0]['data'])
         snapshot =  payload['metrics']['plugins'][0]['data']['snapshot']
         self.assertTrue(snapshot)
+        self.assertIn('m', snapshot)
+        self.assertEqual('Manual', snapshot['m'])
         self.assertIn('version', snapshot)
         self.assertEqual(len(snapshot['versions']), 1)
         self.assertEqual(snapshot['versions']['instana'], VERSION)
+
+
+    @patch.object(HostCollector, "should_send_snapshot_data")
+    def test_prepare_payload_with_autowrapt(self, mock_should_send_snapshot_data):
+        mock_should_send_snapshot_data.return_value = True
+        os.environ["AUTOWRAPT_BOOTSTRAP"] = "instana"
+        self.create_agent_and_setup_tracer()
+
+        payload = self.agent.collector.prepare_payload()
+        self.assertTrue(payload)
+        self.assertIn('snapshot', payload['metrics']['plugins'][0]['data'])
+        snapshot =  payload['metrics']['plugins'][0]['data']['snapshot']
+        self.assertTrue(snapshot)
```

Comment:
Can we have just one assertion here like this?

```suggestion
        snapshot = payload['metrics']['plugins'][0]['data'].get('snapshot')
        self.assertIsNotNone(snapshot)
```

Reply:
While it is indeed one line shorter, and often the shorter the better, this case is trickier.
When the `assertIn` fails, the test framework immediately and clearly communicates to the developer, what is missing from which collection, without having to check the test code.
When the `assertIsNotNone` fails, then it doesn't have a clear meaning without actually checking the test code.
It doesn't show where it is missing from, furthermore it's not even entirely safe to assume that it is missing, theoretically it could also be that it was set to the `None` value.